source/beamline/CLS/CLSSIS3820Scaler.cpp

```diff
@@ -328,7 +333,16 @@ void CLSSIS3820Scaler::onScanningToggleChanged(){
 	emit scanningChanged(startToggle_->withinTolerance(1));
 }
 
-void CLSSIS3820Scaler::onContinuousToggleChanged(){
+void CLSSIS3820Scaler::onContinuousToggleChanged()
+{
+	qDebug() << "\nScaler mode changed.";
```

Comment:
We can keep this until we run some tests on the beamline with scans, but we should remove this before merging it in.